modules/oc-mirror-updating-registry-about.adoc

```diff
@@ -32,7 +32,12 @@ If there are {product-title} releases or Operator versions that you no longer ne
 
 [IMPORTANT]
 ====
-If there are no new or updated images to mirror, the excluded images are not pruned from the target mirror registry. Additionally, if an Operator publisher removes an Operator version from a channel, the removed versions are pruned from the target mirror registry.
+Images are not automatically pruned from the target mirror registry in the following situations:
+
+* If there are no new or updated images to mirror
+* If you are using the Technology Preview OCI feature
+
+Additionally, if an Operator publisher removes an Operator version from a channel, the removed versions are pruned from the target mirror registry.
```

Comment:
Do you need both instances of _Operator_ here? Sounds redundant and a touch awkward to me.

> if a publisher removes an Operator version from a channel
> if an Operator publisher removes a version from a channel

Reply:
I see what you mean, but removing one of them might reduce the clarity some. Since we use the same wording in 4.13+, I'll leave it as is for now.